src/Tmds.Ssh/SshClient.cs

```diff
@@ -676,6 +677,36 @@ private void FreeChannel(uint nr)
             _allocatedChannels[i] = _allocatedChannels[i] & ~mask;
         }
 
+        public async Task<SftpClient> OpenSftpClientAsync(CancellationToken ct)
```

Comment:
Can you move this into it's own file (like SshClient.ExecCommand)?

Reply:
How should I name it? SshClient.SftpSubsystem ?

Reply:
Let's name it `SshClient.Sftp`.